api/src/main/java/io/strimzi/api/kafka/model/kafka/listener/GenericKafkaListenerConfigurationBootstrap.java

```diff
@@ -114,6 +115,19 @@ public void setLoadBalancerIP(String loadBalancerIP) {
         this.loadBalancerIP = loadBalancerIP;
     }
 
+    @Description("External IPs associated to the nodeport service. " + 
+            "These IPs are used by clients external to the kubernetes cluster to access the kafka brokers. " +
```

Comment:
Sorry again but we should use "Kubernetes" with the capital "k".
The same applies to all the others places where you have this `@Description` annotation.

Reply:
As well as Kafka with capital `K`.